sv.c

```diff
@@ -12474,6 +12472,7 @@ Perl_sv_vcatpvfn_flags(pTHX_ SV *const sv, const char *const pat, const STRLEN p
 
 	    uv = PTR2UV(args ? va_arg(*args, void*) : argsv);
 	    base = 16;
+            c = 'x';
```

Comment:
I think it's useful to comment the purpose of this, as I did in my example code.

Reply:
Added the comment, and this comment about the comment. :)